docker/Dockerfile

```diff
@@ -1,72 +1,47 @@
-# use debian as base image
-FROM debian:stretch
+# Use mambaforge as the base image
+# Builds in ~9.5 min and is ~ 2.5 GB on a windows laptop
+FROM mambaorg/micromamba:0.15.3
 
 # Label image
 LABEL \
     "Description"="Container for open source time series InSAR processing with Mintpy" \
     "Github Source"="https://github.com/insarlab/MintPy/" \
     "Installation"="https://github.com/insarlab/MintPy/blob/main/docs/installation.md" \
-    "Dockerfile Author"="Andre Theron" \
-    "Email"="andretheronsa@gmail.com"
+    "Dockerfile Author"="Forrest Williams" \
+    "Email"="forrestfwilliams@icloud.com"
 
-# Install useful programs
-RUN apt-get update && \
-    apt-get install -y \
-    git \
-    python3 \
-    python3-pip \
-    wget \ 
-    vim \
-    zip \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+# Set user name (the micromamba image comes with the micromamba user)
+ARG user=micromamba
```

Comment:
@forrestfwilliams what's your intention with this argument? 

Since this has to be set at build time, anyone pulling the container will get whatever was used. It probably makes the most sense to just leave it as the user that comes with the micromamba image. 

Otherwise, if you expect users to build the container they are likely going to want to specify both the UID and GID as they get mapped directly from host to container, so you'd need add a user with those ids or change the micromamba UID/GID.